functionary/ui/templates/core/task_detail.html

```diff
@@ -1,39 +1,55 @@
 {% load unicorn %}
-
 {% block content %}
-<div class="column is-full">
-    <nav class="breadcrumb has-arrow-separator" aria-label="breadcrumbs">
-        <ul>
-            <li><a href="{% url 'ui:task-list' %}">Task List</a></li>
-            <li class="is-active"><a href="#">{{ task.function.name }}</a></li>
-        </ul>
-    </nav>
-</div>
-<div class="column">
-    {% unicorn 'task_title' task=task %}
-    <div class="column has-addons" >
-        <div class="field">
-            <label class="label"><i class="fa fa-building"></i>&nbsp;{{ task.environment.name }}</label>
-            <label class="label" for="users"><i class="fa fa-user"></i>&nbsp;Creator:</label>
-            <ul id="users">
-                <li class="block ml-4">
-                    <p>{{ task.creator.first_name }} {{ task.creator.last_name }} <span class="has-text-grey-light">({{ task.creator.username }})</span>
-                        at {{ task.created_at }}
-                    </p>
+    <div class="column is-full">
+        <nav class="breadcrumb has-arrow-separator" aria-label="breadcrumbs">
+            <ul>
+                <li>
+                    <a href="{% url 'ui:task-list' %}">Task List</a>
                 </li>
-            </ul>
-        </div>
-        <div class="field">
-            <label class="label" for="users"><i class="fa fa-list"></i>&nbsp;Parameters:</label>
-            <ul id="params">
-                <li class="block ml-4">
-                    <p class="is-font-monospace">{{ task.parameters }}</p>
+                <li class="is-active">
+                    <a href="#">{{ task.function.name }}</a>
                 </li>
             </ul>
+        </nav>
+    </div>
+    <div class="column">
+        {% unicorn 'task_title' task=task %}
+        <div class="column has-addons">
+            <div class="field">
+                <label class="label">
+                    <i class="fa fa-building"></i>&nbsp;{{ task.environment.name }}
+                </label>
+                <label class="label" for="users">
+                    <i class="fa fa-user"></i>&nbsp;Creator:
+                </label>
+                <ul id="users">
+                    <li class="block ml-4">
+                        <p>
+                            {{ task.creator.first_name }} {{ task.creator.last_name }} <span class="has-text-grey-light">({{ task.creator.username }})</span>
+                            at {{ task.created_at }}
+                        </p>
+                    </li>
+                </ul>
+            </div>
+            <div class="field">
+                <label class="label" for="params">
+                    <i class="fa fa-list"></i>&nbsp;Parameters:
+                </label>
+                <ul id="params">
+                    <li class="block ml-4">
+                        <p class="is-font-monospace">{{ task.parameters }}</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="field">
```

Comment:
This file got completely reformatted by djlint, which is why the changeset is somewhat large.  This div is really the only new bit.

I went with the dumbest possible approach here, which just basically replicates the way we're using unicorn for the tasking status.  The problem with this is that there are now two independent polling events to retrieve information that is ultimately derived from the same source (the task).  We should become more familiar with unicorn and figure out how to consolidate these into a single event, but for now I just took the easy way out.

Reply:
I would have updated the endpoint to return the whole task information since there's more than just the status to be updated. This works too and ends quickly enough when the result comes back.

Reply:
Yeah, that probably makes sense.  I worry about how that scales on pages with a lot of data, but you only need to dynamically refresh a couple of independent pieces of it.  In this case, that's not a concern though so I just consolidated the `task_title` and `task_result` components into a single `task_detail` so that there's only a single polling event now.